docs/getting-started/sdk/secrets-manager/integrations/kubernetes.mdx

```diff
@@ -0,0 +1,300 @@
+  self-host scenarios, as well as hitting European servers.
+- **BW_IDENTITY_API_URL** - Sets the Bitwarden Identity service URL that the Secrets Manager SDK
+  uses. This is useful for self-host scenarios, as well as hitting European servers.
+- **BW_SECRETS_MANAGER_STATE_PATH** - Sets the base path where Secrets Manager SDK stores its state
+  files.
+- **BW_SECRETS_MANAGER_REFRESH_INTERVAL** - Specifies the refresh interval in seconds for syncing
+  secrets between Secrets Manager and K8s secrets. The minimum value is 180.
+
+## Running and debugging
+
+1. Install the Custom Resource Definition into the cluster using `make install` or by using the
+   Visual Studio Task called "apply-crd" from the "Tasks: Run Task" in the command palette.
+
+2. To debug the code, just hit F5. You can also use `make run` at the command line to run without
+   debugging.
+
+:::info
+
+You can also run this (without a debugger) in one step by running: `make install run`
+
+:::
+
+### Uninstall Custom Resource Definition
+
+To delete the CRDs from the cluster:
+
+```sh
+make uninstall
+```
+
+:::info
+
+Run `make --help` for more information on all potential `make` targets
+
+:::
+
+### Create a BitwardenSecret to test
+
+With the debugger running, we now will create a BitwardenSecret object to synchronize Secret Manager
+secrets into a K8s secret:
+
+:::caution
+
+Creating the authorization token secret below via kubectl will place your authorization token in the
+machine terminal history. For production systems, consider using a CSI secrets driver or applying
+the secret via an ephemeral build agent.
+
+:::
+
+1. Create a secret to house the Secrets Manager authentication token in the namespace where you will
+   be creating your BitwardenSecret object:
+   `kubectl create secret generic bw-auth-token -n <some-namespace> --from-literal=token="<Auth-Token-Here>"`
+
+2. Install an instances of BitwardenSecret. An example can be found in
+   [config/samples/k8s_v1_bitwardensecret.yaml](https://github.com/bitwarden/sm-kubernetes/blob/main/src/sm-operator/config/samples/k8s_v1_bitwardensecret.yaml)
+   You will want to copy this example and update it for your own needs. Then apply it like so:
+   `kubectl apply -f -n <some-namespace> k8s_v1_bitwardensecret.yaml`
+
+#### BitwardenSecret manifest
+
+Think of the BitwardenSecret object as the synchronization settings that will be used by the
+operator to create and synchronize a Kubernetes secret. This Kubernetes secret will live inside of a
+namespace and will be injected with the data available to a Secrets Manager service account. The
+resulting Kubernetes secret will include all secrets that a specific service account has access to.
+The sample manifest
+([config/samples/k8s_v1_bitwardensecret.yaml](https://github.com/bitwarden/sm-kubernetes/blob/main/src/sm-operator/config/samples/k8s_v1_bitwardensecret.yaml))
+gives the basic structure of the BitwardenSecret. The key settings that you will want to update are
```

Comment:
Would it make sense to have an identifier or something after Bitwarden Secret here, like 'object' or 'operator'?
It reads a little weird to me, but it could just be the standard way of referring to it that I'm not used to.

Reply:
I think BitwardenSecret manifest is correct in this sense, but maybe it should be "The BitwardenSecret manifest".  It is describing the BitwardenSecret Kubernetes manifest structure.  Thoughts? @differsthecat

Reply:
Yes, maybe move it around like `The sample [sample link] gives the basic structure of the BitwardenSecret manifest.`

Reply:
That makes sense.  I actually looked at this incorrectly on Friday.  I see now what you mean.  This has been updated.